algopy_testing/examples/auction/test_contract.py

```diff
@@ -24,9 +24,8 @@ def test_opt_into_asset(context: AlgopyTestContext) -> None:
     contract.opt_into_asset(asset)
 
     # Assert
-    assert len(context.get_transaction_group()) == 1
     assert contract.asa.id == asset.id
-    inner_txn = context.get_last_submitted_inner_transaction()
+    inner_txn = context.get_last_submitted_itxn_loader().asset_transfer()
     assert inner_txn.type == algopy.TransactionType.AssetTransfer
```

Comment:
This one shouldn't be needed right?

Reply:
@robdmoore you mean i shouldn't explicitly know that its an asset transfer and instead to get_last_submitted_itxn_loader().transaction()?